packages/jsActions/nanoflow-actions-native/src/geolocation/GetCurrentLocationMinimumAccuracy.ts

```diff
@@ -5,11 +5,14 @@
 // Other code you write will be lost the next time you deploy the project.
 
 import { Big } from "big.js";
+import { Platform, NativeModules } from "react-native";
 import Geolocation, {
     GeolocationError,
     GeolocationOptions,
-    GeolocationResponse
+    GeolocationResponse,
+    GeolocationStatic
 } from "@react-native-community/geolocation";
+import { GeolocationServiceStatic, GeoError, GeoPosition, GeoOptions } from "../../typings/Geolocation";
```

Comment:
If I understand correctly, you are declaring typings yourself to avoid a static import. 

What about using typescript's `import type` syntax? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

Reply:
I see you used the `type` syntax but kept the typings file - with the `type` keyword, you can import from the dependency that exports these types. Does this work for you?

Reply:
The goal of my suggestion is to remove your user-defined types